### ISSUE TITLE
Make this a PWA (Progressive Web App) for Mobile

### ISSUE THREAD

**Issue by MCBoarder289** [MCBoarder289/shelf-help#4]
It would be cool to make this a PWA and deploy it on phones for more app-like experience. Not sure how it would work with the navigation to other pages, but it's worth trying out.

Here is a helpful video and repo for doing this with Dash Apps:
* [Video Link](https://www.youtube.com/watch?v=aBWqWVBaqhI&t=161s)
* [Repo Link](https://github.com/Coding-with-Adam/Dash-by-Plotly/tree/master/Deploy_App_to_Web/PWA-Phone-App/res-seating)
* [PWA Builder](https://www.pwabuilder.com/)

**Comment by MCBoarder289** [comment]
[Helpful Article - Downloading to home screen](https://medium.com/anuix/how-to-customize-the-add-to-home-screen-banner-for-a-pwa-68f9d1f00830)

**Comment by MCBoarder289** [comment]
While working this, I realized that I may also need to rebrand this app, because the name is certainly not creative enough.

I think I'll rename it `Shelf Help`, and will subsequently update this whole repo and the render webservice.

Other items I want to complete:

- [x] Improve styling:
    - [ ] Center Cards
    - [x] Center Modal
    - [x] Adjust the horizontal padding  on container/gutters (currently: `--bs-gutter-x: 1.5rem;`)
    - [x] Follow dbc best practices (Rows and Cols everywhere)
    - [x] Adjust Shelf URL button to be an [Input Group](https://dash-bootstrap-components.opensource.faculty.ai/docs/components/input_group/) (see DropdownMenu addons)
    - [ ] Nav Items?
- [x] Improve Icons
- [ ] Add Install Instructions
    - [ ] Collapse with Tabs?
    - [ ] Include Gif of install

**Comment by MCBoarder289** [comment]
Will add install instructions on a separate PR. Since I'm rebuilding the render blueprint, this will need to happen first so I can take the video of the install.
